server/public/model/websocket_client.go

```diff
@@ -111,7 +111,7 @@ func makeClient(dialer *websocket.Dialer, url, connectURL, authToken string, hea
 	client.configurePingHandling()
 	go client.writer()
 
-	client.SendMessage(WebsocketAuthenticationChallenge, map[string]any{"token": authToken})
+	client.SendMessage(string(WebsocketAuthenticationChallenge), map[string]any{"token": authToken})
```

Comment:
@agnivade Does it make sense to also change `WebSocketRequest.Action` to a `WebsocketEventType`? Are the events that the client sends to the server similar to the ones the server sends to the client?

Reply:
No it's different. An event is semantically generated from within the server. A websocket action is different.